vms/volunteer/views.py

```diff
@@ -1,256 +1,256 @@
-# standard library

```

Comment:
Extra deletion/insertion showing up. Which editor do you use?

Reply:
Answered above.